Sender_node_can1:
  ros__parameters:
    # 1. kp      0.0 ~ 6553.0      分解能0.1
    # 2. ki      0.0 ~ 6553.0      分解能0.1
    # 3. kd      0.0001 ~ 6.5530   分解能0.0001
    # 4. out_max 100 ~ 25500       分解能100
    # 5. ki_max  100 ~ 6300        分解能100   
    speed_gains_1_can1: [17.0, 200.0, 0.03, 20000.0, 1000.0]
    speed_gains_2_can1: [17.0, 200.0, 0.03, 20000.0, 1000.0]
    speed_gains_3_can1: [17.0, 200.0, 0.03, 20000.0, 1000.0]
    speed_gains_4_can1: [17.0, 200.0, 0.03, 20000.0, 1000.0]
    speed_gains_5_can1: [17.0, 200.0, 0.03, 20000.0, 1000.0]
    speed_gains_6_can1: [17.0, 200.0, 0.03, 20000.0, 1000.0]
    speed_gains_7_can1: [17.0, 200.0, 0.03, 20000.0, 1000.0]
    speed_gains_8_can1: [17.0, 200.0, 0.03, 20000.0, 1000.0]
    position_gains_1_can1: [2000.0, 0.0, 0.5, 2000.0, 1000.0]
    position_gains_2_can1: [2000.0, 0.0, 0.5, 2000.0, 1000.0]
    position_gains_3_can1: [2000.0, 0.0, 0.5, 2000.0, 1000.0]
    position_gains_4_can1: [3000.0, 0.0, 0.5, 2000.0, 1000.0]
    position_gains_5_can1: [2000.0, 0.0, 0.5, 2000.0, 1000.0]
    position_gains_6_can1: [2000.0, 0.0, 0.5, 2000.0, 1000.0]
    position_gains_7_can1: [2000.0, 0.0, 0.5, 2000.0, 1000.0]
    position_gains_8_can1: [2000.0, 0.0, 0.5, 2000.0, 1000.0]
    